seqr/management/commands/check_bam_cram_paths.py

```diff
@@ -21,8 +24,8 @@ def handle(self, *args, **options):
             except Exception as e:
                 individual_id = sample.individual.individual_id
                 failed.append(individual_id)
-                self.stdout.write('Error at {} (Individual: {}): {} '.format(sample.file_path, individual_id, e.message))
+                self.stdout._out.write('Error at {} (Individual: {}): {} \n'.format(sample.file_path, individual_id, str(e)))
 
-        self.stdout.write('---- DONE ----')
-        self.stdout.write('Checked {} samples'.format(len(samples)))
-        self.stdout.write('{} failed samples: {}'.format(len(failed), ', '.join(failed)))
+        self.stdout._out.write('---- DONE ----\n')
```

Comment:
I understand why you made this change based on your explanation, but I think that using protected class members is really bad practice. I like your idea of switching to using logging instead

Reply:
Updated.